tools/sdk-testgen/packages/autorest.gotest/test/integrationtest/output/appplatform/test/zt_generated_spring_live_test.go

```diff
@@ -12,12 +12,9 @@ import (
 	"context"
 	"testing"
 
-	"time"
-
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore/arm"
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
-	"github.com/Azure/azure-sdk-for-go/sdk/internal/recording"
```

Comment:
Hi, does the recording framework removed intentionally?

Reply:
Yes. All the things related with the recording framework have been encapsulated in the internal go package.